cli/src/main.rs

```diff
@@ -133,43 +133,29 @@ fn main() {
             "Could not save minified code"
         );
     } else {
-        let (mut tx, rx) = spmc::channel::<PathBuf>();
-        let mut handles = Vec::new();
-        for _ in 0..num_cpus::get() {
-            let rx = rx.clone();
-            let cfg = cfg.clone();
-            handles.push(thread::spawn(move || {
-                let input = rx.recv().unwrap();
-                let input_name = input.to_string_lossy().into_owned();
-
-                let mut src_file =
-                    io_expect!(input_name, File::open(&input), "Could not open source file");
-                let mut src_code = Vec::<u8>::new();
-                io_expect!(
-                    input_name,
-                    src_file.read_to_end(&mut src_code),
-                    "Could not load source code"
-                );
-                let out_code = minify(&src_code, &cfg);
-                let mut out_file = io_expect!(
-                    input_name,
-                    File::create(&input),
-                    "Could not open output file"
-                );
-                io_expect!(
-                    input_name,
-                    out_file.write_all(&out_code),
-                    "Could not save minified code"
-                );
-            }));
-        }
-
-        for i in args.inputs {
-            tx.send(i).unwrap();
-        }
+        args.inputs.par_iter().for_each(|input| {
+            let input_name = input.to_string_lossy().into_owned();
 
-        for handle in handles {
-            handle.join().unwrap();
-        }
+            let mut src_file =
+                io_expect!(input_name, File::open(&input), "Could not open source file");
+            let mut src_code = Vec::<u8>::new();
+            io_expect!(
+                input_name,
+                src_file.read_to_end(&mut src_code),
+                "Could not load source code"
+            );
+            let out_code = minify(&src_code, &cfg);
+            let mut out_file = io_expect!(
+                input_name,
+                File::create(&input),
+                "Could not open output file"
+            );
+            io_expect!(
+                input_name,
+                out_file.write_all(&out_code),
+                "Could not save minified code"
+            );
+            println!("finish minifying {}", input_name);
```

Comment:
I think it's better to just print the name, since this is the default output and the repeated prefix becomes redundant. It'd also allow piping into another command (quite nice for something like `minify-html *.html | xargs gzip`), copying as list of files, etc.